src/fheroes2/game/game_startgame.cpp

```diff
@@ -789,8 +794,24 @@ fheroes2::GameMode Interface::Basic::HumanTurn( bool isload )
         // hotkeys
         if ( le.KeyPress() ) {
             // if the hero is currently moving, pressing any key should stop him
-            if ( isMovingHero )
+            if ( isMovingHero ) {
                 stopHero = true;
+            }
+
+#if defined( WITH_DEBUG )
+            else if ( HotKeyPressEvent( Game::HotKeyEvent::TRANSFER_CONTROL_TO_AI ) ) {
+                if ( fheroes2::
+                         showMessage( fheroes2::Text( _( "Warning" ), fheroes2::FontType::normalYellow() ),
+                                      fheroes2::Text( _( "Do you want to transfer control from you into the AI? The effect will take place only on the next turn." ),
```

Comment:
The use of "into" is not correct here.
```suggestion
                                      fheroes2::Text( _( "Do you want to transfer control from you to the AI? The effect will take place only on the next turn." ),
```

Reply:
Ah, yes. I missed it.